crates/common/Cargo.toml

```diff
@@ -60,7 +60,7 @@ tokio = "1"
 tracing.workspace = true
 url = "2"
 walkdir = "2"
-yansi = "0.5"
+yansi = "1.0"
```

Comment:
can we move this to workspace deps (the root Cargo.toml) and then use yansi.workspace here?

Reply:
yes I think it would be better cause all crates should share same foundation to keep them as a unity